main.lsf

```diff
@@ -833,11 +833,17 @@ Overlay OVY_59
     After OVY_58
     Object asm/overlay_59.o
 }
-Overlay OVY_60
+Overlay intro_title
 {
     After main
 	Object src/title_screen.o
```

Comment:
inconstistent indentation here

Reply:
the entire file has a mix of tabs and spaces, so fixing it globally to spaces. y'all please change your editor config to indent this file using spaces instead of tabs.

Reply:
will do